src/Smalot/PdfParser/RawData/RawDataParser.php

```diff
@@ -635,14 +635,10 @@ protected function getRawObject(string $pdfData, int $offset = 0): array
                 // name object
                 $objtype = $char;
                 ++$offset;
-                $pregResult = preg_match(
-                    '/^([^\x00\x09\x0a\x0c\x0d\x20\s\x28\x29\x3c\x3e\x5b\x5d\x7b\x7d\x2f\x25]+)/',
```

Comment:
If I understand this part right, you restrict the range from "all string" to "sub string with offset $offset and 256 length". Is that correct?